src/functionalTest/resources/requests/cafcass-cymru-send-email-request.json

```diff
@@ -0,0 +1,311 @@
+              "recipientsOptions": "Jeremy Anderson, Martina Graham",
+              "serveOnRespondent": "No",
+              "cafcassCymruServed": "Yes",
+              "otherPartiesServed": "No",
+              "cafcassOrCymruNeedToProvideReport": "No",
+              "orderEndsInvolvementOfCafcassOrCymru": "No"
+            },
+            "bulkPrintOrderDetails": [
+
+            ],
+            "isTheOrderAboutAllChildren": "No",
+            "isWithdrawnRequestApproved": "",
+            "typeOfChildArrangementsOrder": ""
+          }
+        }
+      ],
+      "serveOrderDynamicList": {
+        "value": [
+          {
+            "code": "87e63f63-c0d8-4593-bfbb-565799a8a63f",
+            "label": "Blank order or directions (C21) - 20 Dec 2023"
+          },
+          {
+            "code": "6ee844b8-8bae-4968-b5d7-6239bdf049ed",
+            "label": "Blank order or directions (C21) - 20 Dec 2023"
+          }
+        ],
+        "list_items": [
+          {
+            "code": "87e63f63-c0d8-4593-bfbb-565799a8a63f",
+            "label": "Blank order or directions (C21) - 20 Dec 2023"
+          },
+          {
+            "code": "6ee844b8-8bae-4968-b5d7-6239bdf049ed",
+            "label": "Blank order or directions (C21) - 20 Dec 2023"
+          }
+        ]
+      },
+      "serveOrderAdditionalDocuments": [
+      ],
+      "serveToRespondentOptions": "Yes",
+      "recipientsOptions": {
+        "value": [
+          {
+            "code": "7663081e-778d-4317-b278-7642b740d317",
+            "label": "John Doe (Applicant 1)"
+          },
+          {
+            "code": "7e9028ba-534c-42bb-bb04-e67b6551b4bd",
+            "label": "Martina Graham (Applicant 3)"
+          }
+        ],
+        "list_items": [
+          {
+            "code": "7663081e-778d-4317-b278-7642b740d317",
+            "label": "John Doe (Applicant 1)"
+          },
+          {
+            "code": "0016cff5-17f0-42af-bea4-0b04ca74c9ec",
+            "label": "Jeremy Anderson (Applicant 2)"
+          },
+          {
+            "code": "7e9028ba-534c-42bb-bb04-e67b6551b4bd",
+            "label": "Martina Graham (Applicant 3)"
+          },
+          {
+            "code": "e406bcc3-3c91-45db-9dcc-3a5c14930851",
+            "label": "Mary Richards (Respondent 1)"
+          },
+          {
+            "code": "8d5140d5-18eb-4278-b41c-2a6fabdf4491",
+            "label": "Elise Lynn (Respondent 2)"
+          },
+          {
+            "code": "62614c4a-4faf-48c0-abfd-c579ef700fd4",
+            "label": "David Carman (Respondent 3)"
+          }
+        ]
+      },
+      "otherParties": {
+        "value": [
+        ],
+        "list_items": [
+          {
+            "code": "6bb5e9ac-df97-4593-8b22-3969dc0bb4e1",
+            "label": "Sam Nolan"
+          }
+        ]
+      },
+      "cafcassCymruServedOptions": "Yes",
+      "cafcassCymruEmail": "khurshed.khan@hmcts.net",
```

Comment:
PLease remove this hmcts email id.

Reply:
@shashisk97 as discussed removed mentioned email id